gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,6 @@
-#Tue Jun 12 21:47:54 IST 2018
+#Wed Mar 20 08:07:11 IST 2019
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-4.4-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-4.10.1-all.zip
```

Comment:
I don't see any harm caused by it as long as the build compiles fine

Reply:
No I am getting errors when running ./gradlew lint pwd checkstyle findbugs.
```
* What went wrong:
Could not determine java version from '11.0.2'.
```
so I had to update gradle version.
reference: https://github.com/openrndr/openrndr/pull/23